tests/infra_test.go

```diff
@@ -996,6 +999,255 @@ var _ = Describe("[Serial]Infrastructure", func() {
+				patchLabels := fmt.Sprintf(`{ "op": "replace", "path": "/metadata/labels", "value": %s}`, string(labelsBytes))
+
+				data := []byte(fmt.Sprintf("[ %s, %s ]", patchTestLabels, patchLabels))
+
+				_, err = virtClient.CoreV1().Nodes().Patch(context.Background(), nodesWithKVM[0].Name, types.JSONPatchType, data, metav1.PatchOptions{})
+				Expect(err).ToNot(HaveOccurred())
+			})
+
+			It("should remove old labeller labels and annotations)", func() {
+				originalNode, err := virtClient.CoreV1().Nodes().Get(context.Background(), nodesWithKVM[0].Name, metav1.GetOptions{})
+				Expect(err).ToNot(HaveOccurred())
+
+				node := originalNode.DeepCopy()
+
+				node.Labels["feature.node.kubernetes.io/cpu-model-Penryn"] = "true"
+				node.Labels["feature.node.kubernetes.io/cpu-feature-mmx"] = "true"
+				node.Labels["feature.node.kubernetes.io/kvm-info-cap-hyperv-synic"] = "true"
+				node.Labels["feature.node.kubernetes.io/some-fancy-feature-which-should-not-be-deleted"] = "true"
+				node.Annotations["node-labeller-feature.node.kubernetes.io/cpu-model-Penryn"] = "true"
+				node.Annotations["node-labeller-feature.node.kubernetes.io/cpu-feature-mmx"] = "true"
+				node.Annotations["node-labeller-feature.node.kubernetes.io/kvm-info-cap-hyperv-synic"] = "true"
+
+				originalLabelsBytes, err := json.Marshal(originalNode.Labels)
+				Expect(err).ToNot(HaveOccurred())
+
+				originalAnnotationsBytes, err := json.Marshal(originalNode.Annotations)
+				Expect(err).ToNot(HaveOccurred())
+
+				labelsBytes, err := json.Marshal(node.Labels)
+				Expect(err).ToNot(HaveOccurred())
+
+				annotationsBytes, err := json.Marshal(node.Annotations)
+				Expect(err).ToNot(HaveOccurred())
+
+				patchTestLabels := fmt.Sprintf(`{ "op": "test", "path": "/metadata/labels", "value": %s}`, string(originalLabelsBytes))
+				patchTestAnnotations := fmt.Sprintf(`{ "op": "test", "path": "/metadata/annotations", "value": %s}`, string(originalAnnotationsBytes))
+				patchLabels := fmt.Sprintf(`{ "op": "replace", "path": "/metadata/labels", "value": %s}`, string(labelsBytes))
+				patchAnnotations := fmt.Sprintf(`{ "op": "replace", "path": "/metadata/annotations", "value": %s}`, string(annotationsBytes))
+
+				data := []byte(fmt.Sprintf("[ %s, %s, %s, %s ]", patchTestLabels, patchLabels, patchTestAnnotations, patchAnnotations))
+
+				_, err = virtClient.CoreV1().Nodes().Patch(context.Background(), nodesWithKVM[0].Name, types.JSONPatchType, data, metav1.PatchOptions{})
+				Expect(err).ToNot(HaveOccurred())
+				time.Sleep(time.Second * 10)
+				node, err = virtClient.CoreV1().Nodes().Get(context.Background(), nodesWithKVM[0].Name, metav1.GetOptions{})
+				Expect(err).ToNot(HaveOccurred())
+
+				foundSpecialLabel := false
+				for key := range node.Labels {
+					Expect(key).ToNot(ContainSubstring("feature.node.kubernetes.io/cpu-model-"), "Node can't contain any label with prefix feature.node.kubernetes.io/cpu-model-")
```

Comment:
Can you extract these strings to constants? They are meaningful and are used in multiple places.